web/src/js/components/ProxyApp.jsx

```diff
@@ -37,6 +38,10 @@ class ProxyAppMain extends Component {
         this.onSettingsChange = this.onSettingsChange.bind(this)
     }
 
+    componentWillMount() {
+        this.props.wsConnect()
```

Comment:
I'm just not comfort with initializing things out of the react - redux architecture. No convincing reason for this :pensive: